public/app/features/alerting/unified/state/actions.ts

```diff
@@ -456,7 +456,7 @@ export const fetchGrafanaNotifiersAction = createAsyncThunk(
 
 export const fetchContactPointsStateAction = createAsyncThunk(
   'unifiedalerting/fetchContactPointsState',
-  (alertManagerSourceName: String): Promise<ContactPointsState> =>
+  (alertManagerSourceName: string): Promise<ContactPointsState | undefined> =>
```

Comment:
dito as above, when would this be `undefined`?

Reply:
ops! true!! I've just pushed the change.